### ISSUE TITLE
Missing label on the close button

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/drive#428]
## Bug Report

**Problematic behavior**
The close button of the "create folder" modal does not have a label

**Expected behavior/code**
The button must have a close label.

**Possible Solution**
Add an attribute `aria-label="Fermer la fenêtre modale"` in the button, knowing that the content of the `aria-label` attribute must be translated into English and Dutch.

**Additional context/Screenshots**

<img width="358" height="226" alt="Image" src="https://github.com/user-attachments/assets/dbc58551-dbfd-4b8d-9d17-46f6746d2809" />
